ci-operator/config/redhat-developer/gitops-operator/redhat-developer-gitops-operator-master.yaml

```diff
@@ -0,0 +1,31 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+promotion:
+  name: "4.7"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+- as: e2e-gitops-operator
+  commands: make test-e2e
+  openshift_installer_src:
+    cluster_profile: aws
```

Comment:
@dewan-ahmed Can you please replace the line from 1-27 with
```yaml
build_root:
  project_image:
    dockerfile_path: openshift-ci/build-root/Dockerfile
resources:
  '*':
    requests:
      cpu: "2"
      memory: 4Gi
tests:
- artifact_dir: /tmp/artifacts
  as: unit
  commands: echo "Call unit test script"
  container:
    from: src
```

Reply:
In the next pr you can go for spinning up the cluster. For now lets onboard the bare minimum openshift CI work flow.

Reply:
@amitkrout thanks. Review feedback implemented via 697be347f3b1d96b0e0d1abdb2f1b28d147c2cc1.